index.bzl

```diff
@@ -0,0 +1,13 @@
+# Copyright Google Inc. All Rights Reserved.
```

Comment:
Google LLC?

Also can use the license banner from our `build-config.js`?

Reply:
Nice catch, thanks. Didn't notice the difference between this repo and angular/angular.